src/module/scripts/template-placement/index.js

```diff
@@ -157,13 +157,12 @@ async function promptMeasureTemplate() {
     return result;
 }
 
-export {
-    addSkipRangeToDialog,
-    promptMeasureTemplate,
-};
+export { addSkipRangeToDialog, promptMeasureTemplate };
 
-const _getSize = (shared) => pf1.utils.convertDistance(RollPF.safeTotal(shared.action.measureTemplate.size, shared.rollData))[0];
-const _getHeight = (shared, distance) => pf1.utils.convertDistance(RollPF.safeTotal(distance, shared.rollData))[0];
+const _getSize = (shared) =>
+    pf1.utils.convertDistance(RollPF.safeRollSync(shared.action.measureTemplate.size, shared.rollData)._total)[0];
+const _getHeight = (shared, distance) =>
+    pf1.utils.convertDistance(RollPF.safeRollSync(distance, shared.rollData)._total)[0];
```

Comment:
Except for this, this entire file seems to have no other changes. Pruning of styles is certainly needed

Reply:
This is repeated across many files